tests/test_symbols.py

```diff
@@ -1146,6 +1146,61 @@ def test_state_size(self):
             self.assertEqual(s.state_size(), 10 * 8)
 
 
+class TestDivide(utils.SymbolTests):
```

Comment:
Should inherit from [BinaryOpTests](https://github.com/dwavesystems/dwave-optimization/blob/3fd8dfa445e7b652df488bc05fd0ac571130edd6/tests/test_symbols.py#L128) unless it doesn't work for the same reason as the C++ tests? In which case we should add a comment.